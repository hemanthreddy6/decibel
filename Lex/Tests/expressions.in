a1*2-b2^0.5^.01[1.:2.2]*(p__1(_, a,b,  d __c   ,_ , -10S+440hz))
1a.3.2
a1. | b || c |||=
