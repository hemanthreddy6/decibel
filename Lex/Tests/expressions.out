IDENTIFIER
*
INT_LITERAL
-
IDENTIFIER
^
FLOAT_LITERAL
^
FLOAT_LITERAL
[
FLOAT_LITERAL
:
FLOAT_LITERAL
]
*
(
IDENTIFIER
(
_
,
IDENTIFIER
,
IDENTIFIER
,
IDENTIFIER
_
_
IDENTIFIER
,
_
,
-
FLOAT_LITERAL
+
FLOAT_LITERAL
)
)
INT_LITERAL
IDENTIFIER
FLOAT_LITERAL
FLOAT_LITERAL
IDENTIFIER
INVALID_SYMBOL
|
IDENTIFIER
||
IDENTIFIER
||
|=
